sprokit/processes/core/serializer_base.cxx

```diff
@@ -113,11 +113,11 @@ ::base_init()
       elem_spec.m_serializer = std::dynamic_pointer_cast< vital::algo::data_serializer > ( base_nested_algo );
       if ( ! elem_spec.m_serializer )
       {
-        std::stringstream str;
-        str << "Unable to create serializer for type \""
+        std::stringstream sstr;
+        sstr << "Unable to create serializer for type \""
```

Comment:
"str" was not a good name anyway, but "sstr" feels odd; does "ss" work? If not, maybe "stream" or "sstream"?